### PR TITLE
[15.0][MIG] website_snippet_marginless_gallery: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT41034

@chienandalu @CarlosRoca13 please review

### CODE REVIEW

File: website_snippet_marginless_gallery/views/snippets.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                        style="background-image: url('/website/static/src/img/library/bridge.jpg');"
+                    >
+                        <div>
+                            <h4 class="o_default_snippet_text">Lorem ipsum</h4>
+                            <p class="o_default_snippet_text mb0">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do.
+                            </p>
+                        </div>
+                    </section>
+                    <section class="col-lg-3 col-12 pt32 pb32 bg-alpha">
```
Comment (chienandalu) [inline website_snippet_marginless_gallery/views/snippets.xml]:
```suggestion
                    <section class="col-lg-3 col-12 pt32 pb32 bg-primary">
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
bg-alpha|beta|delta|gamma et al doesn't exist anymore so the colors in the snippet initial state doesn't show up anymore as they were in the previous version:

image

Yo can use any of the available current classes anyway:

https://github.com/odoo/odoo/blob/15.0/addons/web/static/lib/bootstrap/css/bootstrap.css#L6510-L6596

File: website_snippet_marginless_gallery/views/snippets.xml (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply website_snippet_marginless_gallery/views/snippets.xml]: Class have been updated, thank you :)

**Review by CarlosRoca13 — APPROVED** [review]
Code and functional review :+1:

File: website_snippet_marginless_gallery/views/snippets.xml (hunk 1 above)
Comment (CarlosRoca13) [reply website_snippet_marginless_gallery/views/snippets.xml]: Will we need any migration script for this cases or this migration script will be set on a more generic module?

Comment (pedrobaeza) [reply website_snippet_marginless_gallery/views/snippets.xml]: This is being handled in https://github.com/OCA/openupgradelib/pull/315 and https://github.com/OCA/OpenUpgrade/pull/3707

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**